### ISSUE TITLE
preserve: true outputting some declarations twice.

### ISSUE THREAD

**Issue by kabadesu** [postcss/postcss-calc#7]
If I do this in my source

```
.selector {
  width: calc(100% - 80px);
}
```

The result will be

```
.selector {
  width: calc(100% - 80px);
  width: calc(100% - 80px);
}
```

It should just leave the declaration as-is.

**Comment by kabadesu** [comment]
Sorry I think I've logged this in the wrong place. Should I log it over at [reduce-css-calc](https://github.com/MoOx/reduce-css-calc) instead?

**Comment by MoOx** [comment]
Nope this issue belong to this plugin which is not testing the value before cloning & changing it.

**Comment by MoOx** [comment]
Patch released as 4.0.1
